Shared/BundleExtensions.swift

```diff
@@ -8,6 +8,10 @@ extension Bundle {
     /// Get a string from the info plist dictionary for a given key
     public func infoDictionaryString(forKey key: String) -> String {
         guard let value = object(forInfoDictionaryKey: key) as? String else {
+            if ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"] != nil {
+                /// Using dummy info dictionary while running tests, without host app set.
+                return "com.brave.tests"
```

Comment:
We don't have access to bundle now that Host App is gone, so using fake id here only for tests